lib/new-admin/graphql/schema.js

```diff
@@ -394,6 +395,8 @@ const resolvers = {
       transactions.batch(from, until, limit, offset, deviceId),
     transactionsCsv: (...[, { from, until, limit, offset }]) =>
       transactions.batch(from, until, limit, offset).then(parseAsync),
+    simplifiedTransactionsCsv: (...[, { from, until, limit, offset }]) =>
+      transactions.simplifiedBatch(from, until, limit, offset).then(parseAsync),
```

Comment:
Can't we pass down a "simplified" arg to decide if we want the entire or the simplified version? This will probably get extended in the future and we'll have to add a bunch of different filters.